### PR TITLE
refactor to use poi for demo and bili for publishing to npm

### PR DESCRIPTION
**Pull request by vinaypuppal**
Hi,
> Currently this component is published to NPM as a direct .vue file, which is probably not a good idea. Can someone help me distribute this properly? I have no idea around that.

1. I solved this by using https://egoist.moe/bili/
1. Configured eslint
1. Added MIT License
1. Added `vue`, `vuetify` and `moment` to `peerDependencies`
1. created demo folder which can be build using https://poi.js.org/#/ 
## How to publish to npm?
1. `yarn build` will generate `dist/` folder which can be published to npm which contains these files
	screen shot 2018-02-03 at 18 34 08
1. `yarn publish`

## How to deploy demo folder
1. `yarn build:demo` will generate `build/` folder which can be deployed to `netlify` or `surge.sh`
2. I added https://www.npmjs.com/package/ghpages which can be used to deploy demo folder to `gh-pages`, just run `yarn deploy` it will build demo folder and push `build/` folder to `gh-pages` branch of this repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Vuetify Date Range Picker
 
 > The missing date range picker for Vuetify JS you have been looking for.
-![Date Range Picker](./public/updated-date-range-lres.gif)
+> ![Date Range Picker](https://github.com/praveenpuglia/vuetify-daterange-picker/blob/master/public/updated-date-range-lres.gif)
```
Comment (vinaypuppal) [inline README.md]: We are not publishing `public` folder to npm(see `files` field in `package.json` so I removed the relative link and added GitHub URL for `.gif`)

Comment (praveenpuglia) [reply README.md]: Awesome. Thanks 👍

**Comment by praveenpuglia** [comment]
@vinaypuppal super awesome! Thanks for the help. Really really happy.

One thing though, are you using prettier? If not, can you the run the code through prettier once?

**Comment by vinaypuppal** [comment]
Yes I am using prettier, and also I have https://github.com/vuejs/vetur configured in my editor

File: src/DateRange.vue [hunk 2]
```diff
@@ -14,49 +19,56 @@
...
   </v-card>
 </template>
+
 <script>
-import * as moment from 'moment';
+import moment from 'moment';
+
 export default {
+  props: {
+    options: {
```
Comment (praveenpuglia) [inline src/DateRange.vue]: @vinaypuppal thanks for doing this. I was thinking of doing this next.

Comment (vinaypuppal) [reply src/DateRange.vue]: eslint was complaining before so I added validation for props

**Comment by praveenpuglia** [comment]
Something has screwed up with my vscode installation. Just wanted to test things out.. Need some time.

**Comment by praveenpuglia** [comment]
@vinaypuppal - Merged. Thanks for the awesome PR.

**Comment by vinaypuppal** [comment]
Awesome, let me know once you publish to NPM.

**Comment by praveenpuglia** [comment]
Published! :)

**Comment by vinaypuppal** [comment]
For changelogs or GH releases, you can use this tool https://github.com/zeit/release

**Comment by praveenpuglia** [comment]
Don't forget to ⭐️ the repo. It's your baby now! :) #shameless

**Pull request merged**